example/bwmf/bwmf_task.go

```diff
@@ -393,7 +407,7 @@ func (t *bwmfTask) GetDShard(ctx context.Context, request *pb.Request) (*pb.Resp
 	return resp, nil
 }
 
-func (t *bwmfTask) MetaReady(ctx context.Context, fromID uint64, linkType, meta string) {
+func (t *bwmfTask) MetaReady(ctx context.Context, fromID uint64, meta string) {
```

Comment:
I'm not sure what to support currently.

I'm just fixing the logic here. When you define, say, Parents, are you defining
* the parents of current task, or
* the tasks' parent flag to watch.

It's different. Look at the "master" link type in full_topo as an example.